src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -30,7 +30,10 @@
 
 /**
  * <p>
- * Checks the indentation of the continuation lines in at-clauses.
+ * Checks the indentation of the continuation lines in at-clauses. That is whether the continued
+ * description of at clauses should be intended or not. If the text is not properly intended it
```

Comment:
Please add a sentence describing what a continued description is, which is when the description spans/starts past the line with the tag.

Reply:
Done